conanfile.py

```diff
@@ -30,16 +30,10 @@ def requirements(self):
         if self.options.unitTests:
             self.requires('gtest/1.10.0')
 
-        if self.options.webready and not os_info.is_macos:
-            if os_info.is_windows:
-                self.options['libcurl'].with_ssl = 'darwinssl'
-            else:
-                self.options['libcurl'].with_ssl = 'openssl'
-
         if self.options.xmp:
             self.requires('XmpSdk/2016.7@piponazo/stable') # from conan-piponazo
         else:
-            self.requires('expat/2.3.0')
+            self.requires('expat/2.2.7')
```

Comment:
Long to explain, but 'expat/2.3.0' does not work on windows. So I reverted this change and we keep using 2.2.7